apps/crn-frontend/src/network/users/UserProfile.tsx

```diff
@@ -120,9 +124,13 @@ const UserProfile: FC<UserProfileProps> = ({ currentTime }) => {
             }
           : undefined,
       avatarSaving,
-      sharedOutputsCount: researchOutputsResult.total,
-      upcomingEventsCount: upcomingEventsResult.total,
-      pastEventsCount: pastEventsResult.total,
+      sharedOutputsCount: isUserOnboarded
+        ? researchOutputsResult.total
+        : undefined,
+      upcomingEventsCount: isUserOnboarded
+        ? upcomingEventsResult.total
+        : undefined,
+      pastEventsCount: isUserOnboarded ? pastEventsResult.total : undefined,
```

Comment:
These checks seem slightly redundant since we're already checking at the point of making the request. Is there a reason for doing effectively the same thing twice?

Reply:
So it looks like the reason is to have the tab label be `Upcoming Events` rather than `Upcoming Events (0)`.

Seems a little arbitrary, but fine.

Reply:
🤷‍♂️